src/main/java/Rick.java

```diff
@@ -0,0 +1,144 @@
+        int i = 0;
+
+        while (true) {
+            System.out.print(">");
+            String input = s.nextLine(); //enter string element;
+            input = input.replaceAll("\\s","");//remove all spaces in user input.
+
+
+            if (input.startsWith("bye")) {
+                System.out.println("bye bye! Hope to see you soon (づ￣ ³￣)づ");
+                break;
+            }
+            if (input.startsWith("list")){
+                if (i ==0) {
+                    System.out.println(" No tasks added (;ↀ⌓ↀ)"); //tasks array is empty
+                } else {
+                    System.out.println("Here are the items in your list ᕦ(ò_óˇ)ᕤ :");
+                    for (int j = 0; j < i; j++) {
+                        System.out.print(j + 1+"."); //iterate through tasks array;
+                        System.out.println(tasks[j]);
+
+                    }
+                }
+                continue;
+            }
+            if(input.startsWith("mark")){
+                int taskIndex = 0;//declare and initialise outside try-catch block;
+                try {
+                    taskIndex = Integer.parseInt(input.substring(4)) - 1; //finding the index to mark;
+                    if (taskIndex >= 0 && taskIndex < i) {
+                        tasks[taskIndex].markAsDone();
+                        printDivider();
+                        System.out.println(" Nicee! I've marked this task as done ᕕ( ᐛ )ᕗ :");
+                        System.out.println(tasks[taskIndex]);
+                        printDivider();
+                    } else {
+                        System.out.println("Invalid task number!");
+                    }
+                }catch(NumberFormatException e){
+                    System.out.println("only a number after mark! O_o");
+                }
+                continue;
+            }
+            if(input.startsWith("unmark")){
+                int taskIndex = 0;
+                try {
+                    taskIndex = Integer.parseInt(input.substring(6)) - 1;
+                    if (taskIndex >= 0 && taskIndex < i) {
+                        tasks[taskIndex].markAsUndone();
+                        printDivider();
+                        System.out.println(" Gotcha! I've unmarked this task (≧▽≦)");
+                        System.out.println(tasks[taskIndex]);
+                        printDivider();
+                    } else {
+                        System.out.println("Invalid task number!");
+                    }
+                }catch(NumberFormatException e){
+                    System.out.println("only a number after unmark! o_O");
+                }
+                continue;
+            }
+            if(input.startsWith("todo")){
+                input = input.substring(4);
+                try{
+                if(input.isEmpty()){
+                    throw new RickException();
+                }}catch(RickException e){
```

Comment:
```suggestion
                try {
                    if (input.isEmpty()) {
                        throw new RickException();
                    }
                } catch (RickException e) {
```
Consider following the coding standard as shown above to maintain consistency and readability throughout your code.